hztb-servicemanager/src/main/java/com/svs/hztb/sm/ServiceManager.java

```diff
@@ -59,7 +59,6 @@ protected SpringApplicationBuilder configure(SpringApplicationBuilder applicatio
 
 	@Override
 	public void setServletContext(ServletContext servletContext) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://ec2-52-23-152-124.compute-1.amazonaws.com:9000/coding_rules#rule_key=squid%3AS1186)